tldw_Server_API/tests/DB_Management/test_content_backend_cache.py

```
@@ -169,3 +269,246 @@ def fake_create(cfg):
+    monkeypatch.setattr(watchlists_db, "get_content_backend", lambda _cfg: holder["backend"])
+    monkeypatch.setattr(
+        watchlists_db.WatchlistsDatabase,
+        "ensure_schema",
+        lambda self: schema_calls.append(self._backend_target_key(self._backend) or "<none>"),
+    )
+    monkeypatch.setattr(watchlists_db.WatchlistsDatabase, "_schema_init_keys", set())
+
+    db = watchlists_db.WatchlistsDatabase(user_id="1")
+
+    holder["backend"] = new_backend
+    db.backend
```

Ruff (0.15.10)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)